recipes/mbedtls/all/conanfile.py

```diff
@@ -8,21 +8,18 @@ class MBedTLSConan(ConanFile):
     topics = ("conan", "mbedtls", "polarssl", "tls", "security")
     url = "https://github.com/conan-io/conan-center-index"
     homepage = "https://tls.mbed.org"
-    license = ("GPL-2.0", "Apache-2.0",)
+    license = ("GPL-2.0","Apache-2.0",)
```

Comment:
Why did you remove this space? :D

Reply:
Accident :rofl: I saw it too, when i opened the PR, i did this one on my phone

Reply:
Haha imagine reading this message 20 years ago.
What, coding on your Nokia? 🤣

Reply:
I was a nerdy kid with an old black berry :1st_place_medal:

Reply:
I did it on my ti83